Assets/Scripts/System/Managers/OptionsManager.cs

```diff
@@ -0,0 +1,98 @@
+using UnityEngine;
+using System.Collections;
+using SimpleJSON;
+
+public class OptionsManager : MonoBehaviour {
+
+    //Launcher Options
+    public bool widescreen = true;
+    public enum ForceMode {LOCALONLY, REMOTEONLY, NONE};
+    public ForceMode forceMode = ForceMode.NONE;
+
+    public bool initializing = true;
+
+    //Default Folders
+    public string contentPath = "c:/WINNITRON/Winnitron_Data";
+    public string playlistsPath = "/Playlists";
+    public string musicPath = "/Music";
+    public string attractPath = "/Attract";
+
+	//language
+	public JSONNode language;
+
+    //Sync Settings
+    public JSONNode sync;
+
+    //Options JSON
+    private JSONNode O;
+
+	// Use this for initialization
+    void Awake () {
+        
+        //Load that JSON
+        string optionsFile = Application.dataPath + "/Options/winnitron_options.json";
+        Debug.Log("Loading options from " + optionsFile);
+        var O = GM.data.LoadJson(optionsFile);
+
+        //Widescreen
+        if (O ["launcher"] ["widescreen"] == "false") {
+            widescreen = false;
+        } else {
```

Comment:
There was some Git weirdness, not sure what happened, but if you look at the Develop branch, this bit of code isn't there.  Not sure where it came from.